packages/core/src/boot/init.ts

```diff
@@ -58,14 +56,12 @@ export interface BuildEnv {
   sdkVersion: string
 }
 
-export function commonInit(userConfiguration: UserConfiguration, buildEnv: BuildEnv, isCollectingError: boolean) {
+export function commonInit(userConfiguration: UserConfiguration, buildEnv: BuildEnv) {
```

Comment:
any reason to remove it from it?
passing the errorObservable to rum error collection could have been enough

Reply:
Since the Core error collection is only used by the RUM error collection (in RUM), there is no reason for exposing the error observable in `rum.ts`. It makes things a bit simpler to follow.

`doStartErrorCollection` is following the pattern we've been using to inject dependencies for unit tests, do you think it should be reconsidered?

Reply:
> doStartErrorCollection is following the pattern we've been using to inject dependencies for unit tests

are you referring to `doStartLogs`?

Reply:
Yes

Reply:
IMO, we would probably not need to do that for logs if we split logs.ts like we do with rum.ts.
Anyway, I don't think it qualifies as a pattern if it is used once.

I understand the idea of putting all the error collection stuff at the same place but since we are testing rum error collection without the core error collection, it seems that it is what we want to work with.

By starting the core error collection from rum, we would still pass RUM error collection dependencies as arguments instead of having one imported.

rum.ts responsibility seems already to wire everything together.

I don't have strong opinion on keeping core error collection start in common init or not but it seems more consistent to me if we pass dependencies as arguments as much as possible.

Reply:
> Anyway, I don't think it qualifies as a pattern if it is used once
This has been used multiple times in the past.

Alright, I'll revert my changes.